components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/internal/I18nMgtServiceComponent.java

```diff
@@ -110,6 +112,20 @@ protected void activate(ComponentContext context) {
                 log.error("Error registering Email Template Mgt Service.");
             }
 
+            // Register Notification Template Mgt Service as an OSGi service.
+            NotificationTemplateManagerImpl notificationTemplateManager = new NotificationTemplateManagerImpl();
+            Hashtable<String, String> serviceProperties = new Hashtable<>();
+            serviceProperties.put("service.name", "NotificationTemplateManager");
```

Comment:
Shall we move these to constants?

Reply:
Addressed with https://github.com/wso2-extensions/identity-event-handler-notification/pull/250/commits/b828356ab48d2c21dbdf01a3949e3ec86b98ce14